src/components/CheckoutPriceBox/CheckoutPriceBox.js

```diff
@@ -11,11 +11,14 @@ import {
   StyledPriceBoxWrapper,
   StyledTotalOfferPrice,
   StyledLabel,
-  StyledPriceWrapper
+  StyledPriceWrapper,
+  CouponNoteStyled
 } from './CheckoutPriceBoxStyled';
 
 const CheckoutPriceBox = () => {
-  const { customerPriceInclTax } = useSelector(state => state.offer.offer);
+  const { customerPriceInclTax, period } = useSelector(
+    state => state.offer.offer
```

Comment:
We can use selectOnlyOffer from `offerSlice`
```suggestion
  selectOnlyOffer
```

Reply:
done